### ISSUE TITLE
Update /ver.json route path and format

### ISSUE THREAD

**Issue by pdehaan** [mozilla/fxa-content-server#2864]
There was recently a document circulating about a new version route proposal standard:
https://docs.google.com/document/d/1rGVyiLYvZyKE2oHcSVx-vBmQRKhs1kLLgn7xeCs6qKs/edit

**TL;DR:** Basically we'll need to:
1. Add a `source`  attribute which is the URL for the source repository.
2. Rename the route from `/ver.json` to `/__version__` (or add clever aliases).

---

#### Required Objects
```json
{
    "source": "URL for the source repository",
    "version": "human readable version string",
    "commit": "commit hash"
}
```

#### Optional Objects
Any additional fields as required by the application in JSON format.

#### Definition

| FIELD | DESCRIPTION | REQUIRED | TYPE |
|----|----|:----:|----|
| `source` | Source repository for the code base. | yes | String
| `version` | The human readable version of the code base that has been deployed. | yes | String
| `commit` | The commit hash pointing to deployed code in source control. | yes | String

#### Rationale

- To keep this generic enough to work for all projects and still allow flexibility for what individual devs/project want, we think the 3 required fields are sufficient for the original purpose of this (a consistent location to determine what version of a project is deployed)
- Additional fields can be provided as needed, see some examples in the discussion below 
- Configuration (e.g. puppet) information was purposely excluded because the `__version__` route will be stored with the application source.

**Comment by pdehaan** [comment]
I'm not 100% certain what our `source` would be. I "assume" that it'd be https://github.com/mozilla/fxa-content-server/ but maybe it should be the more generic https://github.com/mozilla/fxa. But I suppose the commit SHA only makes sense in the context of this repo.

**Comment by pdehaan** [comment]
FWIW, here's our current /ver.json file for https://latest.dev.lcip.org/:

```json
{
  "version": "0.42.0",
  "commit": "4b6ab6a97ce52c4b5500590837577430eda570f7",
  "l10n": "ccd87e66f6",
  "tosPp": "27d35cc967"
}
```

Per the spec above, including those additional `l10n` and `tosPp` fields in the top level should be completely valid.

**Comment by rfk** [comment]
> I'm not 100% certain what our source would be.

Indeed, it should be this very repo in order for the commit hash to make sense

**Comment by pdehaan** [comment]
I believe this was fixed by #2915. Closing.
